lib/client/fileUpload.js

```diff
@@ -47,11 +51,18 @@ Template.afFileUpload.helpers({
   },
   uploadedFile() {
     const template = Template.instance();
-    const _id = template.fileId.get() || this.value;
-    if (typeof _id !== 'string' || _id.length === 0) {
-      return null;
-    }
-    return template.collection.findOne({_id:_id});
+    var _id = template.fileId.get() || this.value;
+    if (typeof _id !== 'string' || _id.length === 0) return null;
+    const file = template.collection.findOne({_id:_id});
+    if (!file) return null;
+    file.hasBootstrap = template.hasBootstrap();
+    return file;
+  },
+  getProgress(progress){
+    return parseInt(progress);
+  },
+  hasBootstrap(){
+    return Template.instance().hasBootstrap();
```

Comment:
Shouldn't it be simply?:
```js
return typeof $().emulateTransitionEnd === 'function';
```

Reply:
There is unfortunately no universal method that allows to test if bootstrap is installed, since it can be installed via package or via npm and I found this to be a generally accepted solution on Stackoverflow.

I could test if there is some bootstrap package, that I can import and then return true/false but this may break older installations that do not allow dynamic import.

Reply:
ok just misunderstood, yes I could simply do the line above. I just used the function attached to the template in order to make it easily testable (by just changing the content of the first function in line 29 to false), single point of error.

Reply:
Has been fixed in the latest commit